examples/lifelong_learning/RFNet/basemodel.py

```diff
@@ -0,0 +1,390 @@
+    parser.add_argument('--workers', type=int, default=4,
+                        metavar='N', help='dataloader threads')
+    parser.add_argument('--base-size', type=int, default=1024,
+                        help='base image size')
+    parser.add_argument('--crop-size', type=int, default=768,
+                        help='crop image size')
+    parser.add_argument('--batch-size', type=int, default=6,
+                        help='batch size for training')
+    parser.add_argument('--val-batch-size', type=int, default=1,
+                        metavar='N', help='input batch size for \
+                                    validating (default: auto)')
+    parser.add_argument('--test-batch-size', type=int, default=1,
+                        metavar='N', help='input batch size for \
+                                    testing (default: auto)')
+    parser.add_argument('--num-class', type=int, default=24,
+                        help='number of training classes (default: 24)')
+    parser.add_argument(
+        '--no-cuda',
+        action='store_true',
+        default=False,
+        help='disables CUDA training')
+    parser.add_argument('--gpu-ids', type=str, default='0',
+                        help='use which gpu to train, must be a \
+                            comma-separated list of integers only (default=0)')
+    parser.add_argument('--checkname', type=str, default=None,
+                        help='set the checkpoint name')
+    parser.add_argument(
+        '--weight-path',
+        type=str,
+        default="./models/530_exp3_2.pth",
+        help='enter your path of the weight')
+    parser.add_argument(
+        '--save-predicted-image',
+        action='store_true',
+        default=False,
+        help='save predicted images')
+    parser.add_argument('--color-label-save-path', type=str,
+                        default='./test/color/',
+                        help='path to save label')
+    parser.add_argument('--merge-label-save-path', type=str,
+                        default='./test/merge/',
+                        help='path to save merged label')
+    parser.add_argument('--label-save-path', type=str, default='./test/label/',
+                        help='path to save merged label')
+    parser.add_argument(
+        '--merge',
+        action='store_true',
+        default=False,
+        help='merge image and label')
+    parser.add_argument(
+        '--depth',
+        action='store_true',
+        default=False,
+        help='add depth image or not')
+
+    args = parser.parse_args()
+    args.cuda = not args.no_cuda and torch.cuda.is_available()
+    if args.cuda:
+        try:
+            args.gpu_ids = [int(s) for s in args.gpu_ids.split(',')]
+        except ValueError:
+            raise ValueError(
+                'Argument --gpu_ids must be a comma-separated list of integers only')
+
+    return args
+
+
+def accuracy(y_true, y_pred, **kwargs):
```

Comment:
@luosiqi the `./accuracy.py` has this func `accuracy` in the project, so you can import it.